app/models/integration_collection_mappings/base_collection_mapping.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module IntegrationCollectionMappings
+  class BaseCollectionMapping < ApplicationRecord
+    include PaperTrailTraceable
+    include SettingsStorable
+
+    self.table_name = 'integration_collection_mappings'
+
+    belongs_to :integration, class_name: 'Integrations::BaseIntegration'
+
+    MAPPING_TYPES = descendants.map do |descendant|
```

Comment:
I think that enum definition has to come here, in parent.
Also, probably we need to find a way how to ensure that `.descendants` always return child data in correct order, where new files are added last? We need to ensure correct order so that enum vales don't get mixed up...
That being said, probably we can have MAPPING_TYPES definition only in parent. I think it wouldn't change TBH with other accounting providers...
`MAPPING_TYPES = %i[fallback_item coupon subscription_fee minimum_commitment tax prepaid_credit].freeze`